2.4/Dockerfile

```diff
@@ -38,7 +34,14 @@ RUN yum install -y epel-release && \
     yum clean all && \
     mkdir -p /var/lib/mongodb/data && chown -R mongodb.mongodb /var/lib/mongodb/ && \
     test "$(id mongodb)" = "uid=184(mongodb) gid=998(mongodb) groups=998(mongodb)" && \
-    chmod o+w -R /var/lib/mongodb && chmod o+w -R /opt/rh/mongodb24/root/var/lib/mongodb
+    # Loosen permission bits to avoid problems running container with arbitrary UID
+    chmod -R a+rwx /opt/rh/mongodb24/root/var/lib/mongodb
+
+COPY run-mongod.sh /usr/local/bin/
+COPY contrib /var/lib/mongodb/
+
+# Loosen permission bits to avoid problems running container with arbitrary UID
+RUN chmod -R a+rwx /var/lib/mongodb
```

Comment:
@bparees added this extra RUN line like in the PR for PostgreSQL.
In hindsight, I'm unsure about `a+rwx`, in particular setting the executable bit in every file...
Maybe `a+rw` would be better, leaving the executable bit only for files that where already executable...? But then, would be good to make them executable by all (if they were not already).